### PR TITLE
Feature/roc 9696

### PR DESCRIPTION
**Pull request by jaberelfasi**
**Before creating a pull request make sure that:**

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/ROC-9696

### Change description ###
As a defendant LR

I want to be able to select if I intend to make any applications in future

So that it is captured within the system 

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: src/main/java/uk/gov/hmcts/reform/civil/model/CaseData.java [hunk 1]
```diff
@@ -159,6 +159,8 @@
     private final YesOrNo smallClaimHearingInterpreterRequired;
     private final String smallClaimHearingInterpreterDescription;
 
+    private final String additionalInformationForJudge;
```
Comment (bhssamray) [inline src/main/java/uk/gov/hmcts/reform/civil/model/CaseData.java:162]: Is there a reason why this is not part of new field FutureApplications in Applicant1DQ?

**Review by sampankumar — APPROVED** [review]

File: src/main/java/uk/gov/hmcts/reform/civil/model/dq/Respondent1DQ.java [hunk 2]
```diff
@@ -110,4 +111,10 @@ public WelshLanguageRequirements getWelshLanguageRequirements() {
     public StatementOfTruth getStatementOfTruth() {
         return respondent1DQStatementOfTruth;
     }
+
+    @Override
```
Comment (bhssamray) [inline src/main/java/uk/gov/hmcts/reform/civil/model/dq/Respondent1DQ.java:115]: Please could we try removing 'Override', which probably forces us to put the signature in interface DQ.java, and hence in Applicant1DQ.java

Comment (jaberelfasi) [reply src/main/java/uk/gov/hmcts/reform/civil/model/dq/Respondent1DQ.java]: Done

**Review by bhssamray — DISMISSED** [review]
Demo held and clarification to points commented on.

**Review by sampankumar — APPROVED** [review]

**Review by bhssamray — DISMISSED** [review]

**Review by indu2019 — APPROVED** [review]
Approving it

**Review by bhssamray — APPROVED** [review]

**Review by sampankumar — APPROVED** [review]

**Review by dtortolaV1 — APPROVED** [review]

**Review by indu2019 — APPROVED** [review]
Approved

**Pull request merged**